Upstreamize RHV chapter

lzap opened the pull request:
In addition I am commiting VSCode snippet file. If you use Visual Studio
Code editor for editing adoc files, you can now use `ifsat<TAB>` to
insert snippet quickly. Great time saver, I will add more as I find
those.

One question is on rhv.example.com. I felt like replacing it with
ovirt.example.com is not big deal because it continues with
/ovirt-engine in the URL anyway and I see no point in "hiding" that
OVirt is the upstream project for RHV. We are open source.

I also replaced "My RHV" with "My OVirt" in the compute resource names,
however this can be easily changed to something more neutral like "My
resource".

Other than that this is pretty straightforward conversion, no issues at
all.

rahulbajaj0509 commented on guides/.vscode/asciidoctor.code-snippets:
```diff
@@ -0,0 +1,33 @@
+		],
+		"description": "Macro ifeval with foreman predefined"
+	}
+}
```
Do we need to add a line here?

lzap replied:
Hmm this one was generated by the VSCode. Let me configure it: `files.insertFinalNewline` must be true.

rahulbajaj0509 commented on guides/common/attributes.adoc:
```diff
@@ -15,6 +15,7 @@
 :context: assembly
 
 ifeval::["{build}" == "foreman"]
+:Team: Foreman developers
```
should add this in alphabetical order?

lzap replied:
I've fixed that already, just refresh. I've used GH feature of resolving conflicts and it still says "
This branch cannot be rebased due to conflicts" tho, need to do this locally.

lzap commented:
I rebased.

melcorr commented on guides/doc-Provisioning_Guide/topics/Virt-RHV.adoc:
```diff
@@ -85,14 +107,14 @@ The items listed in this step are provided as Technology Previews. For further i
---description "RHV server at _rhvm.example.com_" \
---url "_https://rhvm.example.com/ovirt-engine/api_" \
+--name "_My_OVirt_" --provider "Ovirt" \
+--description "OVirt server at _ovirt.example.com_" \
```
Could we not just use the variable for {Ovirt} here @lzap would that not just work, rather than making changes?

lzap replied:
That's a good idea.

melcorr replied:
@lzap, all is done except the ovirt.example.com's through the file. If you're too busy, I can do it.

lzap replied:
You say "you're busy" one more time and I quit! ;-) Lemme fix that.

melcorr commented on guides/doc-Provisioning_Guide/topics/Virt-RHV.adoc:
```diff
@@ -101,54 +123,56 @@ To create a Red Hat Virtualization connection, enter the `hammer compute-resourc
 [options="nowrap" subs="+quotes"]
 ----
-# hammer compute-resource image create --name "_Test_RHV_Image_" \
+# hammer compute-resource image create --name "_Test_OVirt_Image_" \
```
Here too, just replace the RHEV with {OVirt} and it should render accordingly?

lzap replied:
Deal.

melcorr commented on guides/doc-Provisioning_Guide/topics/Virt-RHV.adoc:
```diff
@@ -223,9 +247,9 @@ To create a host with image-based provisioning, use the `hammer host create` com
 [options="nowrap" subs="+quotes"]
 ----
-# hammer host create --name "rhv-test2" --organization "_My_Organization_" \
+# hammer host create --name "ovirt-test2" --organization "_My_Organization_" \
```
Why not use the variable?

lzap replied:
Hostnames are always lowercase, we do not enforce this tho. Probably not a problem, it should be converted to lowercase automatically.

melcorr replied:
What I mean is that this is now going to display as ovirt for Satellite also, and I think it should be rhv-test

melcorr commented on guides/doc-Provisioning_Guide/topics/Virt-RHV.adoc:
```diff
@@ -223,9 +247,9 @@ To create a host with image-based provisioning, use the `hammer host create` com
+# hammer host create --name "ovirt-test2" --organization "_My_Organization_" \
 --location "New York" --hostgroup "Base" \
---compute-resource "_My_RHV_" --provision-method image \
+--compute-resource "_My_OVirt_" --provision-method image \
```
Same here

lzap replied:
Yep.

lzap commented:
I rebased, I am going to commit the VSCode snippets directly to master as I want it for other branches.

melcorr commented:
{ovirt}.example.com is the only thing remaining for this PR to merge

lzap commented:
Done, rebased. I've introduced `{ovirt.example.com}` for consistency with `{foreman.example.com}` tho (maybe we will change example.com domain in the future). If you don't mind.

melcorr commented:
Thank you!
